### PR TITLE
Added introspective text to steps where reporting makes sense

### PR DESCRIPTION
**Pull request by pinin4fjords**
This PR adds introspective text to STDOUT for Seurat steps where it seemed to make sense to do so. There were steps like normalisation etc where there wasn't really anything to report.

Where I have reported summaries I've also output parameters (where appropriate) so that different runs with different results can be compared more easily.

There is also a minor bug fix here concerning a default parameter value in seurat-find-clusters.R.

### CODE REVIEW

File: seurat-filter-cells.R [hunk 1]
```diff
@@ -96,6 +97,20 @@ suppressPackageStartupMessages(require(Seurat))
 
 filtered_seurat_object <- FilterCells(seurat_object, subset.names = subset_names, low.thresholds = lt, high.thresholds = ht, cells.use = cells_use)
 
+# Print a summary of the affects of filtering
+
+opt_table <- data.frame(value=unlist(opt), stringsAsFactors = FALSE)[c(-1,-5,-6), , drop = FALSE]
```
Comment (nh3) [inline seurat-filter-cells.R]: It's a bit difficult to see what `c(-1, -5, -6)` excludes, perhaps using something more explicit or adding comment for better readability?

**Review by nh3 — COMMENTED** [review]
Excellent idea to also output parameter. Just a couple of lines perhaps need a bit commenting, otherwise LGTM.

File: seurat-filter-cells.R (hunk 1 above)
Comment (pinin4fjords) [reply seurat-filter-cells.R]: Thanks @nh3 - I've added a commit to address that. Look good?

Comment (nh3) [reply seurat-filter-cells.R]: Look great!

Comment (pinin4fjords) [reply seurat-filter-cells.R]: Awesome- can I get approval then?

**Review by nh3 — APPROVED** [review]
LGTM

**Comment by pinin4fjords** [comment]
Thanks @nh3 !

**Pull request merged**